### DISCUSSION TITLE
custom exporter?

### DISCUSSION THREAD

**Discussion by rikubodor** [dcm4che/dcm4chee-arc-light#4586] [General]
Could you point me to any documentation, tutorials, or examples that provide guidance on writing custom exporter implementations? 
 
Any insights or advice would be greatly appreciated!

**Comment by vrindanayak** [comment]
Take a look at any of these in https://github.com/dcm4che/dcm4chee-arc-light
- dcm4chee-arc-export-curve2pr
- dcm4chee-arc-export-dcm2hl7
- dcm4chee-arc-export-dcm2ups
- dcm4chee-arc-export-dicom
- dcm4chee-arc-export-impax-report
- dcm4chee-arc-export-jivex-report
- dcm4chee-arc-export-noop
- dcm4chee-arc-export-pr2ko
- dcm4chee-arc-export-storage
- dcm4chee-arc-export-stow
- dcm4chee-arc-export-wado
- dcm4chee-arc-export-xdsi

**Comment by rikubodor** [comment]
let me be a bit more specific. Is it possible to write and custom exporter, package it in jar, deploy to custom classpath in wildfly, somehow register it to dcm4chee vna and use it? I would not like to write custom exporter inside vna itself and always build the whole codebase. like does exporter functionality support exporters that are just "plugged in"?
